src/app/components/track-list/track-list.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="toggle-icon" *ngIf="playlistState==='out'" (click)="toggleTracklist()">
```

Comment:
it would be much better not to reassign string values, but manipulate boolean.
1) rename component property to `public isPlaylistClosed = true;`
2) change `*ngIf` checking to `*ngIf="isPlaylistClosed"`
3) rename method `toggleTracklist` to `togglePlaylist` for consistency and change its logic to `this.isPlaylistClosed = !this.isPlaylistClosed;`
4) change `@slideInOut` to `[@slideInOut]="isPlaylistClosed ? 'out' : 'in'"`